.github/workflows/node.js.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x, 17.x]
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
[Currently supported Node versions](https://nodejs.org/en/about/releases/) are 14, 16 and 18, so we should run test for those.